build.sbt

```diff
@@ -8,10 +8,14 @@ lazy val root = (project in file("."))
     scalaVersion := "2.12.8",
     name := "sbt-dependency-from-file",
     organization := "io.radicalbit",
+    description := "sbt-dependency-from-file is a fresh-made library for dynamic load sbt dependencies using json file.",
```

Comment:
change accordingly if you edit here
https://github.com/radicalbit/sbt-dependency-from-file/pull/1/files#diff-04c6e90faac2675aa89e2176d2eec7d8R2